### DISCUSSION TITLE
env GATSBY_EXPERIMENTAL_DEV_SSR=false not working for gatsby@next

### DISCUSSION THREAD

**Discussion by simonjoom** [gatsbyjs/gatsby#30829] [Help]
process.env.GATSBY_EXPERIMENTAL_DEV_SSR = false in my env have no effect and Gatsby still use ssr-static-develop

in webpack.config
Due to processEnv in webpack.config  all Gatsby var are string in output .
so a conditional on process.env.GATSBY_EXPERIMENTAL_DEV_SSR always true 

Solution proposal:
1> change processEnv to format "true" string to boolean (I think the best to avoid this kind of issue in futur)
or just change all conditional process.env.GATSBY_EXPERIMENTAL_DEV_SSR
to process.env.GATSBY_EXPERIMENTAL_DEV_SSR=="true"

**Comment by LekoArts** [comment]
You can disable it in your `gatsby-config.js`:

```js
module.exports = {
  flags: {
    DEV_SSR: false
  }
}
```

**Reply by simonjoom** [reply]
ah ok.
correct in the code DEV_SSR -> put correctly env GATSBY_EXPERIMENTAL_DEV_SSR 

you can close it
